hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/PropertyNames.java

```diff
@@ -228,6 +228,7 @@ private PropertyNames() {
     public static final String HEDERA_RECORD_STREAM_LOG_EVERY_TRANSACTION = "hedera.recordStream.logEveryTransaction";
     public static final String ISS_RESET_PERIOD = "iss.resetPeriod";
     public static final String ISS_ROUNDS_TO_LOG = "iss.roundsToLog";
+    public static final String NETTY_LOGGING_OFF = "netty.loggingOff";
```

Comment:
I do not like the `loggingOff` name. In general setting the property will disable verbose net logging and not reactive the complete net logging. Maybe `netty.disableDebugLogging` is a better name?